app/serializers/trainee_serializer/v01.rb

```diff
@@ -2,22 +2,32 @@
 
 module TraineeSerializer
   class V01
-    EXCLUDE_ATTRIBUTES = %w[
+    EXCLUDED_ATTRIBUTES = %w[
```

Comment:
I think `apply_application_id` is a foreign key which we want to add to this list.

Reply:
@stevehook do you have any objections to add it back to the `EXCLUDED_ATTRIBUTES`? I've removed it as part of https://github.com/DFE-Digital/register-trainee-teachers/pull/4187#discussion_r1560829453